conda/dev.yml

```
@@ -0,0 +1,40 @@
+# Conda development environment for testing local source code changes to `zppy-interfaces` before merging them to production (`master` branch).
+name: zppy-interfaces-dev
+channels:
+  - conda-forge
+  - defaults
+dependencies:
+  # Base
+  # =================
+  - python >=3.9
+  - pip
+  - matplotlib-base
+  - mpas_tools
```

```suggestion
  - mpas_tools >=0.21.0
```
This is the most recent version that changed CIME constants.